### DISCUSSION TITLE
fan air flow

### DISCUSSION THREAD

**Discussion by ogg1e** [mebs/CM4-NAS#36] [General]
How does the air flow through this case? Since the fan is at the bottom, but there's no holes on the bottom, isn't there poor airflow because of that? Wouldn't it make sense to have a large hole in the bottom so it could pull air from the bottom?

Or is the thinking that it has to pull air from the vents on the side to cool the drives and then blow that out the top? So if you did have a hole in the bottom, there'd be no air flow on the drives at all?

**Comment by mebs** [comment]
Correct, the design aims to cool the drives as well as the board. The vents on the sides are air intakes, air then travels down along one side of the drives, through the fan and up the other sides of the drives and on the boards.
It works best if you have all 4 bays populated. If one or more of the bays are empty, I recommend closing them with cardboard from the inside.
You can test it with some smoke like I did here.
https://imgur.com/a/Qxin24q
